### ISSUE TITLE
Random classifier

### ISSUE THREAD

**Issue by chengsoonong** [chengsoonong/crowdastro#31]
Use random classifier instead of logistic regression

**Comment by MatthewJA** [comment]
Instead of a weighted random classifier for benchmarking, could we use a "classifier" that always returns 0 (not a host galaxy)? I think that always returning 0 would do better than a weighted random classifier, so it seems to me that it would make a better benchmark.

**Comment by chengsoonong** [comment]
The key challenge is to figure out a performance metric that allows you to measure goodness. This would make your definition of "better" above more precise. In other words, I claim that accuracy is not a good performance metric.

**Comment by MatthewJA** [comment]
I think I understand what you mean.

Also, scikit-learn has a dummy classifier in `sklearn.dummy.DummyClassifier`, so I've used that. It does slightly better on the raw label prediction problem but considerably worse on the full problem. The relevant notebook is [17_training_with_features](https://github.com/chengsoonong/crowdastro/blob/master/notebooks/17_training_with_features.ipynb).
